dolphin/coordination.py

```diff
@@ -22,14 +22,27 @@
 from tooz import coordination
 from tooz import locking
 
+from dolphin import cryptor
 from dolphin import exception
 from dolphin.i18n import _
 
 LOG = log.getLogger(__name__)
 
 coordination_opts = [
-    cfg.StrOpt('backend_url',
-               default='redis://127.0.0.1:6379',
+    cfg.StrOpt('backend_type',
+               default='redis',
+               help='The back end type to use for distributed coordination.'),
+    cfg.StrOpt('backend_user',
+               default='',
+               help='The back end user to use for distributed coordination.'),
+    cfg.StrOpt('backend_password',
+               default='',
+               help='The back end URL to use for distributed coordination.'),
```

Comment:
Please update help info, details does not look appropriate to config para

Reply:
Done